content/docs/date-picker/changelog/index.um

```diff
@@ -0,0 +1,12 @@
+@inline /templates/sidebar-page.um
+
+@topSection
+  @title Date Picker Changelog
+
+@contentSection
+  @changelogList
+    @groupByApi false
+    @reverseVisibleList true
+    @inline /templates/versions.um
+    @process
+      @inline ../api.um
```

Comment:
We could de-dupe this page by allowing the usage of `inline` inside `describe` blocks - with that change we could do something like this:

```
@inline /templates/changelog-page.um

@changelogContent Date Picker
   @inline ../api.um
```